prow/github/client.go

```diff
@@ -505,15 +505,55 @@ func (c *Client) AddLabel(org, repo string, number int, label string) error {
 	return err
 }
 
+// Use LabelNotFound to indicate that a label is not attached to an issue.
+// For example, removing a label from an issue, when the issue does not have that label.
+type LabelNotFound struct {
```

Comment:
i thought I might have also but I don't see it there. I think I added the file not found error that might be what you are remembering.